pulsar-client/src/main/java/com/yahoo/pulsar/client/api/ConsumerConfiguration.java

```diff
@@ -47,6 +47,12 @@
 
     private long ackTimeoutMillis = 0;
 
+    private int maxRedeliverUnacknowledgedMessagesBatch = 1000;
+
+    private boolean perMessageRedeliverUnacknowledged = false;
```

Comment:
We shouldn't need this flag, the per-message redelivery should always be enabled